src/ODataProvider.cs

```diff
@@ -47,7 +49,7 @@ public class ODataProvider : BaseProvider, ISource, IDestination, IParameterOpti
     #region AddInManager/ConfigurableAddIn Source
 
     [AddInParameter("Predefined endpoint")]
-    [AddInParameterEditor(typeof(GroupedDropDownParameterEditor), "none=true;refreshParameters=true;required=true")]
+    [AddInParameterEditor(typeof(GroupedDropDownParameterEditor), "none=true;refreshParameters=true;required=true;sortBy=,,default")]
```

Comment:
This is related to changes in dw10 and how the sortBy dynamicoption is handled... so it is necessary to do it like this since the , , are empty sortBy options